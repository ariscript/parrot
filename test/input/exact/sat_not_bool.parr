def id(x): x

check:
    true satisfies id,
    2 satisfies id,
    () violates id,
    false satisfies id because 3.

nil
