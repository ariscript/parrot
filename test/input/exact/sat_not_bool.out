while running test at 5:4-5:18, the predicate returned a non-boolean value (2) when called on the left side
while running test at 6:0-6:18, the predicate returned a non-boolean value (()) when called on the left side
while running test at 7:4-7:32, the predicate returned a non-boolean value (3) when called on the explanation
failed test at 7:4-7:32: the left side failed the predicate (false)
check block: 1/4 tests passed
nil
